pixl_imaging/src/pixl_imaging/_orthanc.py

```diff
@@ -99,11 +119,12 @@ async def modify_private_tags_by_study(
                 "Force": True,
                 "Keep": ["StudyInstanceUID", "SeriesInstanceUID", "SOPInstanceUID"],
```

Comment:
We should also use the "Timeout" here as the dicom timeout, hopefully that works